Create amit-prakash.json

iamitprakash opened the pull request:
Adding creater card for myself

# #WeAreDonet Creator Submission Guidelines

* [x] Check all points before submitting the PR.

* [ ] By submitting this pull request to add my creator card, I verify that I adhere to the #WeAreDotnet creator card submission guidelines.

* [ ] Make sure to follow our [Code of Conduct](https://github.com/wearedotnet/dotnet-creator-cards/blob/master/CODE_OF_CONDUCT.md) 

* [ ] We do not accept submissions with the sole intention of promoting a product or company.

* [ ]  We add you as a creator, not your company. 

* [ ]  The moderator team decides whether we accept or reject a submission.

## When Do We Accept Someone's Creator Card Submission? 

* [ ] You must have published at least three content pieces (long form content) since 2022 on any of your channels. 
* .NET itself
* .NET libraries and projects
* .NET languages C#, F#, Visual Basic
* Tools needed during development (for .NET)
* CI/CD
* Only in combination with .NET: SPA Frontend technologies like Angular, Vue.js, React etc. 
* Cloud services related to .NET applications (AWS, Azure...)


* [ ] There must be three content pieces in English. 

* [ ] There needs to be more than tweets. 

* [ ] Longer-form article on LinkedIn is good, but a self-hosted blog or one on a well-known platform like Hashnode, medium etc., is better. 

* [ ] Alternatively, you contributed with another form on a platform like YouTube, Twitch, Udemy or Pluralsight. If you have some recorded talks of conferences or anywhere else, that counts too! So in the form of a recorded video or a book. 

## RSS or Atom Feed 

* [ ] The content with feed.type = dotnet must be .NET related:
* .NET itself
* .NET libraries and projects
* .NET languages C#, F#, Visual Basic
* Tools needed during development (for .NET)
* CI/CD
* Only in combination with .NET: SPA Frontend technologies like Angular, Vue.js, React etc. 
* Cloud services related to .NET applications (AWS, Azure...)

* [ ] the content under feed.type = personal may contain off-topic posts.
* Project management
* leadership
* Other

* [ ] Submit an Atom or RSS URL inside your creator card that belongs to your blog 

* [ ] If you do not submit an RSS or Atom URL. We cannot consider your feed for the aggregation feature. 

## Content Guideline 

* [ ] If you have a self-hosted blog, please ensure you have a valid HTTPS certificate. 

* [ ] We host NO malicious or offensive content on the blog (including photos, swearing, etc.)

* [ ] You must add the full URL to your GitHub profile to your card, so we can assign it to the member account that used the same GitHub account for sign-up.

iamitprakash commented:
can you please merge this PR

matthiasjost commented:
Hey Amit, thanks a lot for your submission.
your LinkedIn newsletter has a lot of memes at first glance in the article images. Somehow that distracted me from going through the articles.
Can you point me to articles about .NET topics?

While checking your submission, I also checked your GitHub profile. You may want to remove the dead link to your homepage and replace the profile photo (it is showing another person in the mirror).

Greetings 

Matthias

iamitprakash commented:
Hi Matthias,

Thanks for reaching out to me.

List of .Net articles are 

1.  [https://www.linkedin.com/pulse/creating-dummy-data-using-bogus-c-amit-prakash ](https://www.linkedin.com/pulse/creating-dummy-data-using-bogus-c-amit-prakash)
2. https://www.linkedin.com/pulse/rate-limiter-net-7-amit-prakash
3. https://www.linkedin.com/pulse/linq-understanding-performance-capabilities-amit-prakash

along with these, i am actively doing my research on newer versions of .Net and this newsletter will be core focused on .Net and sys design

i have removed the dead link from github profile and updated profile picture as well.

Please let me know if you have another input for me as well.

iamitprakash commented:
Hi Matthias,

Thanks for reaching out to me.

List of .Net articles are

1.
https://www.linkedin.com/pulse/creating-dummy-data-using-bogus-c-amit-prakash
<https://www.linkedin.com/pulse/creating-dummy-data-using-bogus-c-amit-prakash>
2. https://www.linkedin.com/pulse/rate-limiter-net-7-amit-prakash
3.
https://www.linkedin.com/pulse/linq-understanding-performance-capabilities-amit-prakash

along with these, i am actively doing my research on newer versions of .Net
and this newsletter will be core focused on .Net and sys design

i have removed the dead link from github profile and updated profile
picture as well.

Please let me know if you have another input for me as well.

*Amit Prakash*
📲
*+91-8101539717*


On Mon, Apr 10, 2023 at 1:13 PM Matthias Jost ***@***.***>
wrote:

> Hey Amit, thanks a lot for your submission.
> your LinkedIn newsletter has a lot of memes at first glance in the article
> images. Somehow that distracted me from going through the articles.
> Can you point me to articles about .NET topics?
>
> While checking your submission, I also checked your GitHub profile. You
> may want to remove the dead link to your homepage and replace the profile
> photo (it is showing another person in the mirror).
>
> Greetings
>
> Matthias
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/WeAreDotnet/dotnet-creator-cards/pull/123#issuecomment-1501509412>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIKA66YQIZQRUDPFQZNVTRDXAO2ZVANCNFSM6AAAAAAWYIOULU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

matthiasjost commented:
Thanks, Amit; I will check again! Greetings matthias

DeeJayTC commented:
Just to add to this here, without an RSS feed we won't show any of your posts on the portal, we do not index LinkedIn posts.
